<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketScreen</class>
 <widget class="QDialog" name="TicketScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ticketHeader">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 18px;</string>
     </property>
     <property name="text">
      <string>[SubUrban Shop]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ticketContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>616</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ticketListWidget"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="accept">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #000000;
    color: #888888;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    font: 14pt &quot;Impact&quot;;
}
QPushButton:hover {
    color: #ffffff;
}
QPushButton:pressed {
    background-color: #111111;
}
QPushButton:disabled {
    background-color: #222222;
    color: #555555;
}</string>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ticketFooter">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Total: $0\nIVA: $0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
